src/test/java/com/connexta/poller/service/TransformPollingServiceUnitTests.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.poller.service;
+
+import org.junit.jupiter.api.Test;
+
+class TransformPollingServiceUnitTests {
```

Comment:
✏️ `TransformPollingServiceTest`

Reply:
I saw a PR yesterday (don't remember which one) that had classes that test classes that ended in "UnitTests". I thought is was something new we started doing. No?